packages/picasso.js/src/core/scales/color/categorical.js

```diff
@@ -12,6 +12,24 @@ const DEFAULT_EXPLICIT_SETTINGS = {
   range: []
 };
 
+const overrideRange = (range, domain, explicit) => {
+  const overridenRange = range.slice();
```

Comment:
Would prefer a name not so similar to the function name. Ex: "newRange", instead of something that is more or less the same.

Reply:
fixed!